go/vt/vtgate/vcursor_impl.go

```diff
@@ -645,6 +690,7 @@ func (vc *vcursorImpl) ExecuteStandalone(ctx context.Context, primitive engine.P
 	// The autocommit flag is always set to false because we currently don't
 	// execute DMLs through ExecuteStandalone.
 	qr, errs := vc.executor.ExecuteMultiShard(ctx, primitive, rss, bqs, NewAutocommitSession(vc.safeSession.Session), false /* autocommit */, vc.ignoreMaxMemoryRows, vc.resultsObserver)
+	vc.logOpTraffic(primitive, qr)
 	return qr, vterrors.Aggregate(errs)
 }
```

Comment:
here logging will happen twice as this calls back into ExecuteMultiShard which will log again.

Reply:
Ignore the above statement